src/ngraph/op/util/opset1.hpp

```diff
@@ -0,0 +1,29 @@
+//*****************************************************************************
```

Comment:
Is `util` the best subdirectory to place this file? I would place it one or 2 levels up.